pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -47,6 +42,20 @@ const (
 	resourceGroup = "cloudpubsubsources.events.cloud.google.com"
 )
 
+// newReconciledNormal makes a new reconciler event with event type Normal, and
+// reason PubSubSourceReconciled.
+func newReconciledNormal(namespace, name string) pkgreconciler.Event {
+	return pkgreconciler.NewEvent(corev1.EventTypeNormal, "Updated", "Updated CloudPubSubSource: \"%s/%s\"", namespace, name)
+}
+
+func newGetPullSubscriptionsWarn(err error) pkgreconciler.Event {
+	return pkgreconciler.NewEvent(corev1.EventTypeWarning, "PubSubSourceFailed", "Getting PubSubSource failed with: %s", err)
+}
+
+func newCreatePullSubscriptionsWarn(err error) pkgreconciler.Event {
+	return pkgreconciler.NewEvent(corev1.EventTypeWarning, "PubSubSourceFailed", "Creating PubSubSource failed with: %s", err)
```

Comment:
PubSubSourceFailed --> PullSubscriptionCreateFailed
and the comment as well: Creating PullSubscription...

so that is clearer what exactly happened

Reply:
done